internal/capability/operand_cleanup.go

```diff
@@ -55,7 +45,7 @@ func operandCleanup(ctx context.Context, opts ...auditOption) auditCleanupFn {
 				// Forcing cleanup of finalizers
 				err = options.client.GetUnstructured(ctx, options.namespace, name, obj)
 				if apierrors.IsNotFound(err) {
-					return nil
+					continue
```

Comment:
Good catch. All of the returns in this loop should probably be a continue, while aggregating the errors into a slice or something.